rocketpy/environment/environment.py

```diff
@@ -1441,6 +1448,11 @@ def process_custom_atmosphere(
                 outputs="Pressure (Pa)",
                 interpolation="linear",
             )
+            self.barometric_height = self.pressure.inverse_function().set_discrete(
+                0, max_expected_height, 1000, extrapolation="constant"
```

Comment:
why 1000 points? this seems to be too much imo. I think 100 points would be accurate enough and already faster